lib/dal/src/migrations/U2412__summary_diagram_update_geometry_insert_row.sql

```diff
@@ -0,0 +1,77 @@
+DROP FUNCTION IF EXISTS summary_diagram_component_update_geometry_v1(
```

Comment:
Isn't it better to use v2 instead of replacing v1?

Reply:
Yes